spacegraphcats/conf/Snakefile

```diff
@@ -185,9 +188,11 @@ rule bcalm_catlas_sort:
     shadow: "shallow"
     params:
         ksize = ksize,
+        seed = cdbg_shuffle_seed if cdbg_shuffle_seed else 42
```

Comment:
[IMO] I haven't tried, but if `cdbg_shuffle_seed` is `0`, then the seed becomes 42. I think the following would be better.

```
seed = cdbg_shuffle_seed if cdbg_shuffle_seed is not None else 42
```

Reply:
excellent point, thank you! fixed in c7b31a7